README.md

```diff
@@ -471,30 +590,47 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
 | Wii U            | ![Wii U](https://img.shields.io/badge/Wii%20U-8B8B8B?style=for-the-badge&logo=wiiu&logoColor=white)                                   | `![Wii U](https://img.shields.io/badge/Wii%20U-8B8B8B?style=for-the-badge&logo=wiiu&logoColor=white)`                                   |
 | Xbox             | ![Xbox](https://img.shields.io/badge/xbox-%23107C10.svg?style=for-the-badge&logo=xbox&logoColor=white)                                | `![Xbox](https://img.shields.io/badge/xbox-%23107C10.svg?style=for-the-badge&logo=xbox&logoColor=white)`                                |
 
+[(Back to top)](#table-of-contents)
+
+### Hashicorp
```

Comment:
Similar to the comment on TOR, think of the categories as 'what they do' rather than 'who they are'.
So, instead of a category titled **Hashicorp** for instance, Vault should be placed in a category like **Cloud Storage** or **Cloud Security**.
Having the categories based on 'who they are' will quickly become clumsy.
I hope this makes sense?